# TA particle parameters

motion_params:
  vel_std:    0.03
  alpha_std:  0.1
  x_std:      0.01
  y_std:      0.01
  theta_std:  0.05
sensor_params:
  hit_std:    2.0
  z_hit:      0.75
  z_short:    0.01
  z_max:      0.07
  z_rand:     0.12
